src/Meshes/geoMesh.ts

```diff
@@ -0,0 +1,790 @@
+import { Vector3, TmpVectors } from "../Maths/math.vector";
+import { Scalar } from '../Maths/math.scalar';
+import { PHI, THIRDR3 } from '../Maths/math.constants';
+import { IsoVector } from '../Maths/math.isovector';
+
+
+
+/**
+ * Class representing data for one face OAB of an equilateral icosahedron
+ * When O is the isovector (0, 0), A is isovector (m, n)
+ * @hidden
+ */
+export class Primary {
+    //properties
+    public _m: number;

```

Comment:
I think you can remove all the leading underscores in property and function names, as the class is hidden anyway.

Reply:
I'm ok with that, but maybe to be compliant with Babylon code writing rules you should add a /** @hidden */ comment in front of each property / function that has a leading underscore and which is public, as it is not meant to be documented/seen by the user.

However, will let @sebavan / @deltakosh decide if that should be enforced or not.

Reply:
Asked about this earlier when I started https://forum.babylonjs.com/t/pr-of-geodesic-and-goldberg-polyhedron-or-not/20707/3?u=johnk

Reply:
I am of course happy to ensure consistency within Babylon.js and will amend anything as decided by the reviewers.

Reply:
Following the convention of `_Exporter` from GLTFExplorer and other classes (`_TimeToken`, `_InternalAbstractMeshDataInfo`) this is ok.
I agree that it will be fine (and probably cleaner) if there will be no `_` before all public elements in this hidden class, but it's fine to leave it this way.

Reply:
2 Babylon.js Typescript experts to 1 student - will delete the _  in hidden classes. :)